src/main/java/org/sonarlint/intellij/SonarLintIntelliJClient.kt

```diff
@@ -121,7 +128,7 @@ object SonarLintIntelliJClient : SonarLintClient {
     }
 
     private fun findInContentRoots(
-        project: Project, fileNames: List<String>
+        project: Project, fileNames: List<String>,
```

Comment:
It was added with the auto-format, I think because I enabled the "Use trailing comma option" as suggested in the [developer-toolset](https://github.com/SonarSource/sonar-developer-toolset#code-style-configuration-for-intellij)